.github/CONTRIBUTING.md

```diff
@@ -94,6 +127,52 @@ mlc docs/
 
 It only checks links within the local docs (it can't check links to other docs sections that are present when we publish all products under https://redocly.com/docs), and doesn't currently check anchors, so take care when renaming pages or titles.
 
+### Built-in rules changes
+
+After adding a new rule, make sure it is added to the `minimal`, `recommended` and `all` rulesets with appropriate severity levels.
```

Comment:
Should we specifically say it should be `off` where it shouldn't be included? What's the process for deciding what goes in which ruleset?

Reply:
Potentially there might be rules that we want to include into, say, recommended ruleset. Also, in the `all` the severity should be `error`. I'll put `off` - `off` - `error` as the defaults here.

Reply:
Sounds good!